.travis.yml

```diff
@@ -33,4 +33,3 @@ deploy:
   on:
     repo: square/certigo
     tags: true
-    go: 1.9
```

Comment:
I am a little curious why this is going away.

Reply:
It was previously set to restrict releases to one particular version, back when we built with multiple Go versions in CI. Not needed anymore since we only build with 1.10 now.

Reply:
👍